versions.json

```diff
@@ -318,6 +318,7 @@
     "qtwinextras": "5.5.1",
     "qtxlsxwriter": "ad90b6a2c21b300138ceb9fe9030a5917230f92d",
     "qtxmlpatterns": "5.5.1",
+    "quazip": "0.7.1",
```

Comment:
You should not update file ` versions.json`. It is done automatically by @MXEBot.